cuml/rf_demo.ipynb

```diff
@@ -0,0 +1,439 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Random Forest Classification\n",
+    "The Random Forest algorithm is a classification algorithm which builds several decision trees, and aggregates each of their outputs to make a prediction. This makes it more robust to overfitting. The RandomForestClassifier model implemented in the cuML library can accept the following parameters : \n",
+    "1. n_estimators: (default = 10) number of trees in the forest.\n",
```

Comment:
Ok, this section is a nice improvement. I think it's probably a bit more detailed than needed. I'd change it to something like, "You can consult RandomForestClassifier docstrings to see all of the parameters, but here some of the most important ones are: " (then add the text you already have for max_estimators, max_depth, n_bins, deleting the rest.)